module/VuFindSearch/src/VuFindSearch/Backend/EDS/Backend.php

```diff
@@ -137,6 +137,8 @@ class Backend extends AbstractBackend
      */
     protected $isGuest;
 
+    protected $backendType = null;
```

Comment:
TODO: add comment to property and associated setter method.

Reply:
Added